package.json

```diff
@@ -13,6 +13,7 @@
     "start:scheduler": "node schedule.js",
     "start:scheduler:prod": "NODE_ENV=production node schedule.js",
     "setup": "node scripts/setup.js",
+    "pretest": "./node_modules/eslint/bin/eslint.js ./test/*.js ./src/*.js ./services/*.js",
```

Comment:
I prefer the linting to run after the tests, as I get frustrated by my tests not being run because of esthetics when trying out things.

I suggest to create a `lint` npm script and either add it to the CI commands, or add it at the end of the `test` script with `&&` 🙂